### ISSUE TITLE
wheel build fails - no error message

### ISSUE THREAD

**Issue by tonkolviktor** [databrickslabs/dbx#624]
Hello,

Thanks for the development it's in heavy use at our side :)

We have the following problem:
- our configs and setup was tested during 0.7.* times
- now developers started using 0.8.* (0.8.7) and our dbx deploy fails
- the big problem is that the actual stderr and stdout of the build is not printed (which is good normally, but not in case of an error)
- thus we have no way to know what to do

![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/4534324/204995802-de3758ee-3fd8-4947-80d1-96db6122e38a.png)
![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/4534324/204995806-633bc66e-a96e-4390-bac3-614d6c57bfa1.png)

## Expected Behavior
- print all related error files (focus of this ticket)
(- ideally do the build without error :))

## Current Behavior
- no error details visible

## Steps to Reproduce (for bugs)
- ?
- we have a pretty normal pip package

## Your Environment

* dbx version used: 0.8.7
* Databricks Runtime version: 10.4 LTS

**Comment by copdips** [comment]
could you please run directly `/use/bin/python3 -m pip wheel -w dist -e . --prefer-binary --no-deps`, and copy the error message ?

**Comment by tonkolviktor** [comment]
thanks for your answer. the error comes on my collegue machine and he's out for the week.

However that's also kind of not too relevant, my goal by opening this ticket was to ask you to **print** the stderr/out.

More specifically https://github.com/databrickslabs/dbx/blob/8dc436d7d1f9604583735f13f2fb2bf94ff5e8f0/dbx/api/build.py#L28
This message is not too helpful, if you could do something like: 
`dbx_echo("\n💥Command execution failed, \nstdout:{exc.stdout} \nstderr:{exc.stderr}")`

**Comment by copdips** [comment]
> thanks for your answer. the error comes on my collegue machine and he's out for the week.
> 
> However that's also kind of not too relevant, my goal by opening this ticket was to ask you to **print** the stderr/out.
> 
> More specifically
> 
> https://github.com/databrickslabs/dbx/blob/8dc436d7d1f9604583735f13f2fb2bf94ff5e8f0/dbx/api/build.py#L28
> 
> 
> This message is not too helpful, if you could do something like:
> `dbx_echo("\n💥Command execution failed, \nstdout:{exc.stdout} \nstderr:{exc.stderr}")`

yes, this would be helpful, at least my pypi password is masked by the stdout.
